File: modulo1/segmento3/apunte7Funciones/ejercicio6.py
def sumar(num1,num2):
	total=0
	total = num1 + num2
	return total

def restar(num1,num2):
	total=0
	total = num1 - num2
	return total

def dividir(num1,num2):
	total=0
	total = num1 / num2
	return total

def multiplicar(num1,num2):
	total=0
	total = num1 * num2
	return total

def limpiar():
	os.system('clear')

def potencia(num1,num2):
	total=num1
	for i in range(1,num2):
		total*=num1
	
	return total
#main
import os
num1=0
num2=0
opcionMenu=''
while opcionMenu !="0":
	print("1-Suma")
	print("2-Resta")
	print("3-Dividir")
	print("4-Multiplicacion")
	print("5-Potencia")
	print("")
	opcionMenu=input("Ingrese la opcion deseada: ")
	
	if opcionMenu == "1":
		limpiar()
		num1=int(input("Ingrese un numero: "))
		num2=int(input("Ingrese un numero: "))
		total=sumar(num1,num2)
		print("El resultado de la suma es: " + str(total))
	elif opcionMenu == "2":
		limpiar()
		num1=int(input("Ingrese un numero: "))
		num2=int(input("Ingrese un numero: "))
		total=restar(num1,num2)
		print("El resultado de la resta es: " + str(total))
	elif opcionMenu == "3":
		limpiar()
		num1=int(input("Ingrese un numero: "))
		num2=int(input("Ingrese un numero: "))
		total=dividir(num1,num2)
		print("El resultado de la division es: " + str(total))
	elif opcionMenu == "4":
		limpiar()
		num1=int(input("Ingrese un numero: "))
		num2=int(input("Ingrese un numero: "))
		total=multiplicar(num1,num2)
		print("El resultado de la multiplicacion es: " + str(total))
	elif opcionMenu == "5":
		limpiar()
		num1=int(input("Ingrese un numero: "))
		num2=int(input("Ingrese el exponente: "))
		total=potencia(num1,num2)
		print("El resultado de la potencia es: " + str(total))

File: modulo1/segmento3/apunte7Funciones/Ejercicio6.java
import java.util.Scanner;

public class Ejercicio6{

	public static void clearScreen() {  
		System.out.print("\033[H\033[2J");  
		System.out.flush();  
	}  
	public static Integer potencia(Integer num, Integer potencia){

		Integer total=num;

		for(Integer i=1;i<potencia;i++){
			total*=num;
		}

		return total;
	
	
	}

	public static Integer sumar(Integer number1, Integer number2){
		Integer total;

		total = number1+number2;

		return total;
	}

	public static Integer restar(Integer number1, Integer number2){
		Integer total;

		total = number1-number2;

		return total;
	}
	public static Integer multiplicar(Integer number1, Integer number2){
		Integer total;

		total = number1*number2;

		return total;
	}
	
	public static Integer division(Integer number1, Integer number2){
		Integer total;

		total = number1/number2;

		return total;
	}


	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);
		Integer number1,number2;
		Integer total;
		String opcionMenu="1";

		while(!opcionMenu.equals("0")){
			System.out.println("1-Sumar ");
			System.out.println("2-Restar ");
			System.out.println("3-Multiplicar ");
			System.out.println("4-Division ");
			System.out.println("5-Potencia ");
			System.out.println("0-Salir ");
			System.out.println("Ingrese la opcion deseada: ");		

			opcionMenu=teclado.nextLine();

			switch(opcionMenu){
				case "1":
					clearScreen();
					System.out.print("Ingrese un numero: ");
					number1 = Integer.parseInt(teclado.nextLine());
					System.out.print("Ingrese un numero: ");
					number2 = Integer.parseInt(teclado.nextLine());
					total=sumar(number1,number2);
					System.out.println("El total de la suma es: "+ total);
					break;
				case "2":
					clearScreen();
					System.out.print("Ingrese un numero: ");
					number1 = Integer.parseInt(teclado.nextLine());
					System.out.print("Ingrese un numero: ");
					number2 = Integer.parseInt(teclado.nextLine());
					total=restar(number1,number2);
					System.out.println("El total de la resta es: "+ total);
					break;
				case "3":
					clearScreen();
					System.out.print("Ingrese un numero: ");
					number1 = Integer.parseInt(teclado.nextLine());
					System.out.print("Ingrese un numero: ");
					number2 = Integer.parseInt(teclado.nextLine());
					total=multiplicar(number1,number2);
					System.out.println("El total de la multiplicacion es: "+ total);
					break;
				case "4":
					clearScreen();
					System.out.print("Ingrese un numero: ");
					number1 = Integer.parseInt(teclado.nextLine());
					System.out.print("Ingrese un numero: ");
					number2 = Integer.parseInt(teclado.nextLine());
					total=division(number1,number2);
					System.out.println("El total de la division es: "+ total);
					break;	
				case "5":
					clearScreen();
					System.out.print("Ingrese un numero: ");
					number1 = Integer.parseInt(teclado.nextLine());
					System.out.print("Ingrese un numero: ");
					number2 = Integer.parseInt(teclado.nextLine());
					total=potencia(number1,number2);
					System.out.println("El total es: "+ total);
					break;		
				default:
					clearScreen();
					System.out.println("Ingrese una opcion correcta ");
					break;	
			}
		}	
	
	}

}
